src/qutip_qip/qiskit/converter.py

```diff
@@ -0,0 +1,175 @@
+from qutip_qip.circuit import QubitCircuit
+from qiskit.quantum_info import Operator
+import numpy as np
+from qutip import Qobj
+from typing import Union
+
+from qiskit.circuit import QuantumCircuit, Instruction
+
+_map_gates = {
+    "p": "PHASEGATE",
+    "x": "X",
+    "y": "Y",
+    "z": "Z",
+    "h": "SNOT",
+    "s": "S",
+    "t": "T",
+    "rx": "RX",
+    "ry": "RY",
+    "rz": "RZ",
+    "swap": "SWAP",
+}
+
+_map_controlled_gates = {
+    "cx": "CX",
+    "cy": "CY",
+    "cz": "CZ",
+    "crx": "CRX",
+    "cry": "CRY",
+    "crz": "CRZ",
+    "cp": "CPHASE",
+}
+
+_ignore_gates = ["id", "barrier"]
+
+
+def _make_user_gate(unitary: np.ndarray, instruction: Instruction):
```

Comment:
Let's make it clear that it is the qiskit Instracution, `qutip` also has its own `Instruction`.